mbed/mbed.py

```diff
@@ -753,7 +753,7 @@ def publish(all_refs=None):
 
     def fetch():
         info("Fetching revisions from remote repository to \"%s\"" % os.path.basename(getcwd()))
-        popen([git_cmd, 'fetch', '--all', '--tags'] + (['-v'] if very_verbose else ([] if verbose else ['-q'])))
```

Comment:
Could you comment on why `--all` is being dropped for both `git push` and `git fetch`? Is it fixing a bug? Is it an optimization?